server/src/main/java/org/apache/druid/server/coordinator/duty/KillUnusedSegments.java

```diff
@@ -127,7 +148,7 @@ public DruidCoordinatorRuntimeParams run(final DruidCoordinatorRuntimeParams par
     if (canDutyRun()) {
       return runInternal(params);
     } else {
-      log.debug(
+      log.info(
```

Comment:
I think this log can get quite chatty too if the autoKill duty period set to a more continous value, like 30 seconds or 1 minute.

Reply:
Hm, this is an exit condition I think is worth keeping. Also, there's a similar log in the `MarkOvershadowedSegmentsAsUnused` which by default runs every minute (more frequent than the default 30m `KillUnusedSegments` duty cycle): https://github.com/apache/druid/blob/master/server/src/main/java/org/apache/druid/server/coordinator/duty/MarkOvershadowedSegmentsAsUnused.java#L74

I will remove it if we feel strongly about it.

Reply:
Okay, I will downgrade the newly added info logs for termination conditions to debug.

Reply:
Thanks!!